src/IO/InterfaceImpl.cpp

```diff
@@ -1,4 +1,4 @@
-#include "config/config.h"
+
```

Comment:
I don't think this should be removed

Reply:
Good point. I guess the linter can't recognize defines.